conans/test/unittests/tools/env/test_env.py

```diff
@@ -201,6 +201,7 @@ def test_env_files():
                "MyPath3": "OldPath3",
                "MyPath4": "OldPath4",
                }
+    prevenv.update(dict(os.environ.copy()))
```

Comment:
Note for reviewers: some environment variables, like PATH are necessary for the ps1 test. If they don't exist, the powershell.exe command will not be found or will exit with errors

Reply:
Yes, but this could be affecting the previous test functionality. I think it is better to decouple the PS test then, and leave the previous .bat and .sh without requiring this. Or make it cleaner with a parameterized test.

Reply:
Done